eng/Tools.props

```diff
@@ -1,45 +1,15 @@
 <Project>
-  <PropertyGroup>
-    <MSBuildTreatWarningsAsErrors>false</MSBuildTreatWarningsAsErrors>
```

Comment:
We probably still want the TreatWarningsAsErrors right? In order to catch errors where one of the packages you were trying to restore wasn't available so you got a different one instead.

Reply:
Right but this was set to false before

Reply:
Ohh right! haha my bad.